### ISSUE TITLE
The latest version does not work on Chrome

### ISSUE THREAD

**Issue by LinhTran1** [frappe/gantt#89]
The following error is thrown when I initialize a new Gantt
> Invalid argument passed for element
    at Gantt.setup_wrapper (frappe-gantt.js:948)
    at new Gantt (frappe-gantt.js:934)

Error caused by the following code
`
        if (!(element instanceof HTMLElement)) {
            throw new Error('Invalid argument passed for element');
        }
`

The element parameter was a svg element and certainly it is an instance of SVGElement however library just accepts an instance of HTMLElement. Anyone else encounter this error also?

**Comment by Adioud** [comment]
I had this error: since the new version you just have to give a "div" as parameter, not an svg.
The sample page is out of date.

**Comment by LinhTran1** [comment]
I tried with div element however not successful. Nothing was rendered

**Comment by jamieday** [comment]
Can you provide more details? I have no problems running this in chrome with the following code:

HTML:
```HTML
<div class="gantt-target"></div>
```

JS:
```JavaScript
var gantt_chart = new Gantt(".gantt-target", tasks)
```

**Comment by bazzacad** [comment]
Can you please update your home page: https://frappe.io/gantt
I'm just getting started with frappe-gantt & had the same issue.
I also had to include the .css, which wasn't specified.

![2018-07-05 13_52_39-frappe_ frameworks apps](https://user-images.githubusercontent.com/23222121/42347528-0008bb76-805b-11e8-98c3-1c9967b1f555.png)

**Comment by alperyazgan** [comment]
svg element is not working, if it is given with a div it renders with black background ?

**Comment by netchampfaris** [comment]
Sorry for the problems caused. I did not realize the element parameter was not working as before.
Fixed in latest [v0.2.0](https://github.com/frappe/gantt/commit/4b11841bc1b3f4d591e87c1af079ec8c8f0e9098)
Will also update the docs.
